swizzle/README.md

```diff
@@ -10,8 +10,25 @@ $ python provision.py
 ```
 
 Ansible:
-```
-$ ansible-playbook -i $inventory swizzle.yml <extra ansible args>
-```
+
+Assumes you have machines up and ready to use for your cluster.  They need to be either Debian or Red Hat derivatives.  Nothing need be installed besides the base OS.
+
+1. Create an inventory file and edit it to reflect your infrastructure.
+
+    ```
+    $ cp inventory.ini.example inventory.ini
+    ```
+
+2. Create an extra vars file and edit to contain the values that will work for your cluster and infrastructure.  The values contained in the example are the default values so you can delete those you don't wish to change or that don't apply to your installation.
+
+    ```
+    $ cp extra_vars.yaml.example extra_vars.yaml
+    ```
+
+3. Run the playbook.
+
+    ```
+    $ ansible-playbook -i inventory.ini swizzle.yml --extra-vars "@extra_vars.yaml"
```

Comment:
I am getting the following error when trying to run ansible-playbook:

```
ERROR! the role 'common' was not found in /Users/abrand/go/src/github.com/heptiolabs/wardroom/swizzle/roles:/Users/abrand/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/Users/abrand/go/src/github.com/heptiolabs/wardroom/swizzle

The error appears to have been in '/Users/abrand/go/src/github.com/heptiolabs/wardroom/swizzle/swizzle.yml': line 12, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  roles:
    - common
      ^ here
```

Do I need to run this from the wardroom/ansible dir?

Reply:
I had to set the following env var to get it to find the roles:
```
ANSIBLE_ROLES_PATH=../ansible/roles ansible-playbook
```

Reply:
Yes, you will need to run from the `wardroom/ansible` directory. It is possible to override this behavior by setting the DEFAULT_ROLES_PATH via ansible configuration: https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-configuration-settings

Reply:
Ah, just noticed the symlink :)

Reply:
I included a link to the roles directory in this commit to avoid this problem but the environment variable is a more ansible-idomatic way to do this.

Reply:
Updated the instructions to use the env var method and removed the symlink.  Thanks @alexbrand

Reply:
Awesome, thanks @lander2k2!